src/home/home.scss

```diff
@@ -175,15 +175,16 @@
       column-gap: 10%;
       font-family: Inter;
 
-      // .pic {
-      //   img {
-      //     width: 450px;
-      //     height: 450px;
-      //   }
-      // }
+      .pic {
+        img {
+            border-radius: 6px;
+          // width: 450px;
+          // height: 450px;
+        }
+      }
 
       .rightColumn {
-        width: 32em;
+        width: 100%;
```

Comment:
can you please try instead of this changing line 74 to:

`grid-template-columns: auto 1fr;`

this is the first choice for where to control the layout.  Be sure to check how the mobile sizes behave with this change.